deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -618,3 +618,63 @@ cephObjectStores:
       #     - objectstore.example.com
       #   secretName: ceph-objectstore-tls
       # ingressClassName: nginx
+
+# cephECBlockPools are disabled by default, please remove the comments and set desired values to enable it
+#cephECBlockPools:
+#  - name: replicated-metadata-pool
```

Comment:
It's confusing at first why a replicated pool is needed under the EC block pools section. How about adding a comment here that explains a replicated metadata pool is necessary for erasure coding.